ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -3595,6 +3612,7 @@ CHDXSC:
   - compressor_speed_frequency_sensor
   - compressor_speed_percentage_command
   - compressor_speed_percentage_sensor
+  - oil_compressor_pressure_sensor
```

Comment:
prefer to make a new abstract type for oil monitoring using the existing `oil_pressure_sensor` field, and remove the field from this type

Reply:
Removed the compressor oil pressure sensor as not able to model as per the requirement.